src/components/standard/article.tsx

```diff
@@ -103,10 +104,13 @@ const Standard = ({ item, children }: Props): JSX.Element => {
                 {children}
             </Body>
             {epicContainer}
-            <footer css={articleWidthStyles}>
+            <section id="tags" css={articleWidthStyles}>
```

Comment:
What does this `id` get used for?

Reply:
It's used to position epics above I think, so I'll change this to js-tags classname